### PR TITLE
refactor(config): docker POST_OFFICE to use EMAIL_BACKEND

### PR DESCRIPTION
**Pull request by macintoshpie**
#### Any background context you want to provide?
ClearlyEnergy integrated PostOffice into SEED.

#### What's this PR do?
- uses EMAIL_BACKEND for POST_OFFICE configuration instead of logging emails to console.

#### How should this be manually tested?
After merge into develop, roll the web and celery pods, then send an email and verify it is sent properly

#### What are the relevant tickets?
#2728

### CODE REVIEW

**Comment by macintoshpie** [comment]
@aviveiros11 can you confirm we use `config.settings.docker` for our k8s deployment?

**Comment by aviveiros11** [comment]
> @aviveiros11 can you confirm we use `config.settings.docker` for our k8s deployment?

@macintoshpie  Yep, that's correct.  Note: this also pulls in everything from `common.py` as well.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -27,9 +27,11 @@ jobs:
         uses: actions/cache@v2
         with:
           path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          # using `-v2` in key to clear old cache due to errors
+          # See: https://stackoverflow.com/questions/63521430/clear-cache-in-github-actions
+          key: ${{ runner.os }}-buildx-v2-${{ github.sha }}
           restore-keys: |
-            ${{ runner.os }}-buildx-
+            ${{ runner.os }}-buildx-v2-
```
Comment (macintoshpie) [inline .github/workflows/ci.yml:34]: [NPM was failing for strange reasons](https://github.com/SEED-platform/seed/runs/2975148999?check_suite_focus=true) -- this busts the cache and starts over fresh

**Review by Ryoken — APPROVED** [review]
Looks good

**Pull request merged**